cmake/modules/Findlibusb.cmake

```diff
@@ -137,8 +108,29 @@ else ()                                                         # all other OS (
 
     FIND_PACKAGE_HANDLE_STANDARD_ARGS(libusb DEFAULT_MSG LIBUSB_INCLUDE_DIR LIBUSB_LIBRARY)
     mark_as_advanced(LIBUSB_INCLUDE_DIR LIBUSB_LIBRARY)
-
     if (NOT LIBUSB_FOUND)
-        message(FATAL_ERROR "libusb library not found on your system! Install libusb 1.0.x from your package repository.")
+        message(STATUS "No libusb-1.0 not installed into your system. Downloading and building it from source")
```

Comment:
This is not a good practise from my point of view.
All unix-based systems should continue to install and use `libusb `provided from their respective package repositories. It must be considered a very bad attempt to mix locally installed newer library versions with probably already existing dependencies used by other (system) packages originating from the package system, as this opens the door for incompatibilities and functional failures.

Reply:
Ok, no problem, Will revert to the old practice! 

As for windows, Will do some checks for dll locations compatibility later today or tmr, so we can be sure for this as well